key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -665,7 +682,9 @@ varexpId: // weigl, 2021-03-12: This will be later just an arbitrary identifier.
 
 varexp_argument
 :
-    sortId //also covers possible varId
```

Comment:
Ambiguity: `x` might be a name of a sort, or a variable. 

Either we resolve this programmatically during the  taclet building or by a new identifier, e.g., `\varcond(\new(\sort(x)))`.

Reply:
I think the parentheses `\sort(x)` are not in the grammar

Is every word in sortId also in term? Then we could drop this here and have this decided programmatically (as it probably is right now)